### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by iartemenko**
Add Qemu based minimal image target containing aos-vis.

Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: recipes-core/systemd/systemd_%.bbappend [hunk 1]
```diff
@@ -0,0 +1 @@
+PACKAGECONFIG_append = " networkd"
```
Comment (al1img) [inline recipes-core/systemd/systemd_%.bbappend]: Remove it. It is included by default.

File: recipes-core/images/aos-image-minimal.bb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+SUMMARY = "A small image which containing aos-vis."
```
Comment (al1img) [inline recipes-core/images/aos-image-minimal.bb]: Change it to: "An image which contains AOS components"

File: recipes-core/images/aos-image-minimal.bb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+IMAGE_LINGUAS = " "
+IMAGE_FSTYPES = "wic.vmdk"
+LICENSE = "MIT"
+
+inherit core-image
+
+IMAGE_ROOTFS_SIZE ?= "8192"
+IMAGE_ROOTFS_EXTRA_SPACE_append = "${@bb.utils.contains("DISTRO_FEATURES", "systemd", " + 4096", "" ,d)}"
+
+IMAGE_INSTALL_append = "aos-vis tzdata"
```
Comment (al1img) [inline recipes-core/images/aos-image-minimal.bb]: why we need tzdata?

File: recipes-core/images/aos-image-minimal.bb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+IMAGE_LINGUAS = " "
+IMAGE_FSTYPES = "wic.vmdk"
+LICENSE = "MIT"
+
+inherit core-image
+
+IMAGE_ROOTFS_SIZE ?= "8192"
+IMAGE_ROOTFS_EXTRA_SPACE_append = "${@bb.utils.contains("DISTRO_FEATURES", "systemd", " + 4096", "" ,d)}"
+
```
Comment (al1img) [inline recipes-core/images/aos-image-minimal.bb]: Also add following variable:

WIC_CREATE_EXTRA_ARGS = "--no-fstab-update"

File: recipes-core/systemd/systemd_%.bbappend (hunk 1 above)
Comment (iartemenko) [reply recipes-core/systemd/systemd_%.bbappend]: Done

File: recipes-core/images/aos-image-minimal.bb (hunk 2 above)
Comment (iartemenko) [reply recipes-core/images/aos-image-minimal.bb]: Done

File: recipes-core/images/aos-image-minimal.bb (hunk 4 above)
Comment (iartemenko) [reply recipes-core/images/aos-image-minimal.bb]: Done

**Pull request merged**